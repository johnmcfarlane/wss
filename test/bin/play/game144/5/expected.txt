 96 11,1  | GALERES
 84 11,1  | REGALES
 51 11,1  | GALERE
 48 11,1  | GREASE
 46  8,6  | LUGS
 44  8,6  | RUGS
 40 11,2  | ALEGER
 40 11,2  | ARGLES
 40 11,2  | EAGLES
 40 11,2  | GLARES
 40 11,2  | LARGES
 39 11,1  | ELRAGE
 39 11,1  | REGALE
 38 11,3  | GAELS
 37 11,2  | AGREES
 37 11,2  | EAGRES
 37 11,3  | GALES
 37 11,3  | GLEES
 37 11,3  | LAGER
 37 11,3  | LEGER
 37 11,3  | LEGES
 37 11,2  | RAGEES
 37 11,2  | SAGEER
 34 11,3  | AEGER
 34 11,3  | EAGER
 34  8,2  | GALERUS
 34 11,3  | GREES
 34 11,2  | LEASER
 34  8,6  | LUG
 34 11,3  | RAGES
 34 11,2  | REALES
 34 11,3  | REGES
 34 11,1  | RESEAL
 34 11,3  | SAGER
 34 11,2  | SEALER
 34 11,3  | SEGAR
 33 11,1  | ALERSE
 33 11,1  | RESALE
 32 11,3  | EARLS
 32 11,3  | REALS
 32 11,3  | REELS
 32  8,6  | RUG
 31 11,2  | ARGEL
 31 11,3  | ARLES
 31 11,2  | ERGAL
 31 11,3  | LARES
 31 11,3  | LASER
 31 11,3  | RALES
 31 11,2  | REGAL
 31 11,2  | REGEL
 31 11,3  | RELES
 31 11,3  | SLEER
 30 11,2  | AEGLE
 30 11,2  | AGLEE
 30 11,2  | ARGLE
 30 11,2  | EAGLE
 30 11,2  | GALEE
 30 11,2  | GARLE
 30 11,2  | GLARE
 30 11,2  | LARGE
 30  8,4  | LARUS
 30 11,2  | REGLE
 28 11,3  | EASER
 28 11,3  | EGAL
 28 11,3  | GAEL
 28 11,3  | GEAL
 28  8,7  | URSAL
 27 11,2  | AGREE
 27 11,2  | EAGRE
 27 11,3  | GALE
 27 11,2  | GARSE
 27 11,2  | GEASE
 27 11,3  | GLEE
 27 11,3  | LEGE
 27 11,2  | RAGEE
 27 11,2  | SARGE
 27 11,2  | SERGE
 27  8,7  | URSAE
 26  8,5  | ALUR
 26  8,7  | URSA
 25 11,2  | AREEL
 25 11,2  | EASEL
 25 11,2  | SERAL
 25  8,7  | URS
 24 11,3  | AGEE
 24 11,2  | ARSLE
 24 11,2  | EARLE
 24 11,3  | EGRE
 24 11,3  | GARE
 24 11,3  | GERE
 24 11,3  | GREE
 24 11,2  | LEASE
 24 11,3  | RAGE
 24 11,3  | SAGE
 24 11,2  | SLARE
 22 11,3  | EARL
 22 11,3  | ERAL
 22  8,6  | LUR
 22 11,3  | REAL
 22 11,3  | REEL
 22 11,3  | SEAL
 22 11,3  | SEEL
 21 11,3  | ALEE
 21 11,3  | ARLE
 21 11,3  | ELSA
 21 11,3  | ELSE
 21 11,2  | ERASE
 21 11,4  | GALS
 21  2,6  - GARSIL
 21 11,4  | GELS
 21  2,6  - GRESIL
 21 11,3  | LARE
 21 11,3  | LASE
 21 11,3  | LERE
 21 11,3  | LESE
 21 11,3  | RALE
 21 11,3  | SALE
 21 11,2  | SAREE
 21 11,3  | SELE
 21 11,3  | SLAE
 21 11,3  | SLEE
 20 11,4  | GLAR
 20  8,6  | RUS
 20  8,6  | SUR
 19 11,4  | AGER
 19 11,4  | AGES
 19 11,4  | EELS
 19 11,4  | EGER
 19 11,4  | GAES
 19 11,4  | GEAR
 19 11,4  | GEER
 19 11,4  | GEES
 19  7,10 - GLARE
 19 11,4  | GRAS
 19 11,4  | GRES
 19 11,4  | RALS
 18 11,4  | ALES
 18 11,5  | ALS
 18 11,3  | ARSE
 18 11,3  | EASE
 18  2,6  - EGERIA
 18 11,5  | ELS
 18 11,3  | ERSE
 18 11,5  | GAR
 18 11,5  | GAS
 18 11,5  | GES
 18 11,4  | LEAR
 18 11,4  | LEAS
 18 11,4  | LEER
 18  1,6  - LEERSIA
 18 11,4  | LEES
 18 11,3  | RASE
 18 11,3  | SARE
 18 11,3  | SERA
 18 11,3  | SERE
 17 11,4  | ARES
 17 11,4  | ERAS
 17  8,10 - GARE
 17 11,5  | LAR
 17  8,1  | LARGE
 17 11,5  | LAS
 17 11,5  | LER
 17 11,4  | REES
 17  2,6  - RESAIL
 17 11,4  | SEAR
 17 11,4  | SEER
 17  2,6  - SERAIL
 17  7,10 - SLARE
 16 11,5  | AER
 16 11,5  | AES
 16 11,5  | EAR
 16 11,5  | EER
 16  2,6  - GALEI
 16  8,10 - LARE
 16  8,2  | LEGE
 16 11,5  | RAS
 16 11,5  | RES
 16  8,9  | SAG
 16  8,1  | SARGE
 16  8,9  | SEG
 16 11,5  | SER
 16  8,1  | SERGE
 16  1,6  - SERGEI
 15  8,1  | AEGLE
 15 11,6  | AR
 15  8,1  | ARGLE
 15 11,6  | AS
 15  8,1  | EAGLE
 15 11,6  | ER
 15 11,6  | ES
 15  8,1  | GARLE
 15  8,2  | RAGE
 15  8,10 - RARE
 15  8,1  | REGLE
 15  8,2  | SAGE
 15  8,9  | SAL
 15  8,10 - SARE
 15  8,9  | SEL
 15  8,9  | SLA
 14  8,3  | AGE
 14  9,10 - ARE
 14  8,2  | GALE
 14  9,5  | NATAKA
 14  8,9  | SAE
 14  8,9  | SEA
 14  8,9  | SEE
 14  8,9  | SER
 13  8,1  | AGLEE
 13  8,1  | ARSLE
 13  8,5  | ARU
 13  8,1  | EARLE
 13  8,5  | ESU
 13 11,4  | GAL
 13  8,1  | GALEE
 13 11,4  | GEL
 13  8,1  | GLARE
 13  8,8  - SAB
 13  8,7  | US
 12  8,1  | AGREE
 12  8,2  | ARLE
 12  8,1  | EAGRE
 12  4,5  - ESQ
 12  8,1  | GARSE
 12  8,1  | GEASE
 12  8,2  | GLEE
 12  8,6  - LAWS
 12  8,1  | RAGEE
 12  8,2  | RALE
 12  8,2  | SALE
 12  7,6  | SARGE
 12  8,2  | SELE
 12  4,5  - SEQ
 11  7,7  | AEGLE
 11 11,4  | AGE
 11  8,2  | AGEE
 11  8,3  | ALE
 11  7,7  | ALGAE
 11  7,7  | ALGAS
 11  7,7  | ARGAL
 11  7,7  | ARGEL
 11  3,6  - ARGIL
 11  7,7  | ARGLE
 11 11,4  | EEL
 11  8,2  | EGRE
 11  8,3  | ELA
 11 11,4  | GAE
 11  8,2  | GARE
 11 11,4  | GEE
 11  8,2  | GERE
 11 11,4  | GRA
 11  3,6  - GRAIL
 11 11,4  | GRE
 11  8,2  | GREE
 11  8,6  - LAW
 11  8,1  | LEASE
 11  8,6  | LU
 11 11,4  | REL
 11  8,3  | RLE
 11  7,6  | SAGEER
 11 11,4  | SAL
 11 11,4  | SEL
 11  8,9  - SKEF
 11  8,3  | SLA
 11  8,1  | SLARE
 10  7,7  | AEGER
 10  7,7  | AGLEE
 10 11,5  | AL
 10 11,4  | ALE
 10  8,2  | ALEE
 10  7,7  | ALGA
 10  7,7  | ARGAS
 10 11,5  | EL
 10 11,4  | ELA
 10  8,2  | ELSA
 10  8,2  | ELSE
 10  8,1  | ERASE
 10  8,3  | GAE
 10  4,6  - GAIL
 10  8,3  | GEE
 10  8,3  | GRA
 10  8,3  | GRE
 10  8,4  | LA
 10  8,2  | LARE
 10  8,2  | LASE
 10 11,4  | LEA
 10 11,4  | LEE
 10  8,2  | LERE
 10  8,2  | LESE
 10 11,4  | RLE
 10  7,6  | SAGER
 10  7,6  | SAGRA
 10  8,1  | SAREE
 10  2,6  - SERAI
 10 11,4  | SLA
 10  8,2  | SLAE
 10  8,2  | SLEE
  9  7,7  | AGRAL
  9  7,7  | ALG
  9 11,4  | ARE
  9  8,2  | ARSE
  9  8,2  | EASE
  9 11,4  | ERA
  9 11,4  | ERE
  9  8,2  | ERSE
  9 11,4  | ESE
  9  5,6  - GIL
  9  4,6  - GLIA
  9 11,5  | LA
  9  8,3  | LEA
  9  8,3  | LEE
  9  8,2  | RASE
  9 11,4  | REA
  9 11,4  | REE
  9  3,6  - REGIA
  9 11,4  | SAE
  9  7,6  | SAGA
  9  7,6  | SAGE
  9  7,6  | SALAR
  9  8,2  | SARE
  9 11,4  | SEA
  9 11,4  | SEE
  9  8,2  | SERA
  9  8,2  | SERE
  8 11,5  | AE
  8  7,7  | AGAL
  8  7,7  | AGARS
  8  7,7  | AGERS
  8  7,7  | AGREE
  8  7,7  | ARAGE
  8  8,3  | ARE
  8  7,7  | ARG
  8  4,6  - ARIL
  8  7,7  | ARLES
  8 11,5  | EE
  8  8,3  | ERA
  8  8,3  | ERE
  8  8,3  | ESE
  8  4,6  - RAIL
  8 11,5  | RE
  8  8,3  | REA
  8  8,3  | REE
  8  8,3  | SAE
  8  4,6  - SAIL
  8  7,6  | SALA
  8  7,6  | SALE
  8  7,6  | SAREE
  8  8,3  | SEA
  8  8,3  | SEE
  7  7,7  | AALS
  7  8,4  | AE
  7  7,7  | AGAR
  7  7,7  | AGAS
  7  7,7  | AGEE
  7  7,7  | AGER
  7  7,7  | AGES
  7  5,6  - AIL
  7  7,7  | AREAL
  7  7,7  | AREEL
  7  7,7  | AREG
  7  7,7  | ARLE
  7  7,7  | ARSLE
  7  7,7  | ASALE
  7  9,6  - AWL
  7  8,4  | EE
  7  4,6  - ELIA
  7  3,6  - GALI
  7  8,4  | RE
  7  7,6  | SAG
  7  7,6  | SARE
  7  6,7  - SAUTER
  7  6,7  - SAUTES
  7  5,6  - SIL
  6  7,7  | AAL
  6  7,7  | AGA
  6  7,7  | AGE
  6  7,7  | AGR
  6  7,7  | ALAE
  6  7,7  | ALAR
  6  7,7  | ALAS
  6  7,7  | ALEA
  6  7,7  | ALEE
  6  7,7  | ALES
  6  7,7  | ARASE
  6  7,7  | AREAE
  6  7,7  | AREAS
  6  9,6  - AWA
  6  9,6  - AWE
  6  4,6  - ERIA
  6  3,6  - RAGI
  6  7,6  | SAL
  5  7,7  | ALA
  5  7,7  | ALE
  5  7,7  | ALS
  5  7,7  | AREA
  5  7,7  | ARES
  5  7,7  | ARSE
  5  7,7  | ASEA
  5  3,6  - LARI
  5  3,6  - LASI
  5  5,6  - RIA
  5  7,6  | SAA
  5  7,6  | SAE
  5  5,6  - SIA
  4  7,7  | AAS
  4  7,7  | AER
  4  7,7  | AES
  4  7,7  | AG
  4  7,7  | ARE
  4  7,7  | ARS
  4  4,6  - ELI
  4  5,6  - GI
  4  4,6  - LAI
  4  4,6  - LEI
  4  3,6  - SARI
  4  3,6  - SERI
  3  7,7  | AL
  3  8,5  - AN
  3  8,5  - EN
  3  5,6  - LI
  3  4,6  - SAI
  3  4,6  - SEI
  3  4,6  - SRI
  2  7,7  | AA
  2  7,7  | AE
  2  5,6  - AI
  2  7,7  | AR
  2  7,7  | AS
  2  5,6  - SI
