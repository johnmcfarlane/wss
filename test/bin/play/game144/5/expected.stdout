 96 11,1  | GALERES
 84 11,1  | REGALES
 51 11,1  | GALERE
 48 11,1  | GREASE
 40 11,2  | ARGLES
 40 11,2  | EAGLES
 40 11,2  | GLARES
 40 11,2  | LARGES
 39 11,1  | REGALE
 37 11,2  | AGREES
 37 11,2  | EAGRES
 37 11,3  | GALES
 37 11,3  | GLEES
 37 11,3  | LEGES
 37 11,2  | RAGEES
 34 11,3  | GREES
 34 11,2  | LAREES
 34  8,6  | LUG
 34 11,3  | RAGES
 34 11,2  | REALES
 34 11,3  | REGES
 34 11,1  | RESEAL
 33 11,1  | RESALE
 32 11,3  | EARLS
 32 11,3  | REALS
 32 11,3  | REELS
 32  8,6  | RUG
 31 11,3  | ARLES
 31 11,3  | LARES
 31 11,3  | RALES
 31 11,2  | REGAL
 30 11,2  | AGLEE
 30 11,2  | ARGLE
 30 11,2  | EAGLE
 30 11,2  | GLARE
 30 11,2  | LARGE
 28 11,3  | EGAL
 27 11,2  | AGREE
 27 11,2  | EAGRE
 27 11,3  | GALE
 27 11,3  | GLEE
 27 11,2  | RAGEE
 27 11,2  | SARGE
 27 11,2  | SERGE
 25 11,2  | EASEL
 25 11,2  | SERAL
 24 11,3  | AGEE
 24 11,3  | GREE
 24 11,2  | LAREE
 24 11,2  | LEASE
 24 11,3  | RAGE
 24 11,3  | SAGE
 22 11,3  | EARL
 22 11,3  | REAL
 22 11,3  | REEL
 22 11,3  | SEAL
 22 11,3  | SEEL
 21 11,3  | ALEE
 21 11,3  | ELSE
 21 11,2  | ERASE
 21 11,4  | GALS
 21 11,4  | GELS
 21 11,3  | LASE
 21 11,3  | RALE
 21 11,3  | SALE
 21 11,2  | SAREE
 19 11,4  | AGES
 19 11,4  | EELS
 19 11,4  | GAES
 19 11,4  | GEES
 18 11,4  | ALES
 18 11,5  | ALS
 18 11,3  | ARSE
 18 11,3  | EASE
 18 11,5  | ELS
 18 11,5  | GAS
 18 11,4  | LEAS
 18 11,4  | LEES
 18 11,3  | RASE
 18 11,3  | SERA
 18 11,3  | SERE
 17 11,4  | ARES
 17 11,4  | ERAS
 17  8,1  | LARGE
 17 11,5  | LAS
 17 11,4  | REES
 17  2,6  - RESAIL
 17  2,6  - SERAIL
 16 11,5  | RAS
 16 11,5  | RES
 16  8,1  | SARGE
 16  8,1  | SERGE
 15  8,1  | ARGLE
 15 11,6  | AS
 15  8,1  | EAGLE
 15 11,6  | ES
 15  8,2  | RAGE
 15  8,2  | SAGE
 14  8,3  | AGE
 14  8,2  | GALE
 13  8,1  | AGLEE
 13 11,4  | GAL
 13 11,4  | GEL
 13  8,1  | GLARE
 13  8,8  - SAB
 13  8,7  | US
 12  8,1  | AGREE
 12  8,1  | EAGRE
 12  4,5  - ESQ
 12  8,2  | GLEE
 12  8,1  | RAGEE
 12  8,2  | RALE
 12  8,2  | SALE
 12  7,6  | SARGE
 11 11,4  | AGE
 11  8,2  | AGEE
 11  8,3  | ALE
 11  7,7  | ALGAE
 11  7,7  | ALGAS
 11  7,7  | ARGAL
 11  3,6  - ARGIL
 11  7,7  | ARGLE
 11 11,4  | EEL
 11 11,4  | GAE
 11 11,4  | GEE
 11  3,6  - GRAIL
 11  8,2  | GREE
 11  8,1  | LAREE
 11  8,1  | LEASE
 11 11,4  | SAL
 11 11,4  | SEL
 10  7,7  | AGLEE
 10 11,5  | AL
 10 11,4  | ALE
 10  8,2  | ALEE
 10  7,7  | ALGA
 10 11,5  | EL
 10  8,2  | ELSE
 10  8,1  | ERASE
 10  8,3  | GAE
 10  8,3  | GEE
 10  8,4  | LA
 10  8,2  | LASE
 10 11,4  | LEA
 10 11,4  | LEE
 10  7,6  | SAGER
 10  8,1  | SAREE
 10  2,6  - SERAI
  9 11,4  | ARE
  9  8,2  | ARSE
  9  8,2  | EASE
  9 11,4  | ERA
  9 11,4  | ERE
  9 11,4  | ESE
  9  4,6  - GLIA
  9 11,5  | LA
  9  8,3  | LEA
  9  8,3  | LEE
  9  8,2  | RASE
  9 11,4  | REE
  9 11,4  | SAE
  9  7,6  | SAGA
  9  7,6  | SAGE
  9 11,4  | SEA
  9 11,4  | SEE
  9  8,2  | SERA
  9  8,2  | SERE
  8 11,5  | AE
  8  7,7  | AGARS
  8  7,7  | AGERS
  8  7,7  | AGREE
  8  8,3  | ARE
  8  4,6  - ARIL
  8  7,7  | ARLES
  8 11,5  | EE
  8  8,3  | ERA
  8  8,3  | ERE
  8  8,3  | ESE
  8  4,6  - RAIL
  8 11,5  | RE
  8  8,3  | REE
  8  8,3  | SAE
  8  4,6  - SAIL
  8  7,6  | SALE
  8  7,6  | SAREE
  8  8,3  | SEA
  8  8,3  | SEE
  7  7,7  | AALS
  7  8,4  | AE
  7  7,7  | AGAR
  7  7,7  | AGAS
  7  7,7  | AGEE
  7  7,7  | AGER
  7  7,7  | AGES
  7  5,6  - AIL
  7  7,7  | AREAL
  7  9,6  - AWL
  7  8,4  | EE
  7  8,4  | RE
  7  7,6  | SAG
  7  6,7  - SAUTES
  6  7,7  | AAL
  6  7,7  | AGA
  6  7,7  | AGE
  6  7,7  | AGS
  6  7,7  | ALAE
  6  7,7  | ALAR
  6  7,7  | ALAS
  6  7,7  | ALEE
  6  7,7  | ALES
  6  7,7  | AREAE
  6  7,7  | AREAS
  6  9,6  - AWA
  6  9,6  - AWE
  6  3,6  - RAGI
  6  7,6  | SAL
  5  7,7  | ALA
  5  7,7  | ALE
  5  7,7  | ALS
  5  7,7  | AREA
  5  7,7  | ARES
  5  7,7  | ARSE
  5  7,7  | ASEA
  5  3,6  - LARI
  5  5,6  - RIA
  5  7,6  | SAE
  4  7,7  | AAS
  4  7,7  | AG
  4  7,7  | ARE
  4  7,7  | ARS
  4  5,6  - GI
  4  4,6  - LEI
  4  3,6  - SARI
  3  7,7  | AL
  3  8,5  - AN
  3  8,5  - EN
  3  5,6  - LI
  3  4,6  - RAI
  3  4,6  - SEI
  3  4,6  - SRI
  2  7,7  | AA
  2  7,7  | AE
  2  5,6  - AI
  2  7,7  | AR
  2  7,7  | AS
  2  5,6  - SI
